deployment/terraform/dns.tf

```diff
@@ -0,0 +1,48 @@
+#
+# Private DNS resources
+#
+resource "aws_route53_zone" "internal" {
+  name       = "${var.r53_private_hosted_zone}"
+  vpc_id     = "${module.vpc.id}"
+  vpc_region = "${var.aws_region}"
+
+  tags {
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_route53_record" "database" {
+  zone_id = "${aws_route53_zone.internal.zone_id}"
+  name    = "database.service.${var.r53_private_hosted_zone}"
+  type    = "CNAME"
+  ttl     = "10"
+  records = ["${module.database.hostname}"]
+}
+
+#
+# Public DNS resources
+#
+resource "aws_route53_zone" "external" {
+  name = "${var.r53_public_hosted_zone}"
```

Comment:
I'm not sure what the project DNS status is. I think it's likely they'll want to manage that at some point, but I'm inclined to leave it as is for now and adjust later if necessary.

Managing outside terraform makes sense in that case, thanks for noting that.